src/app/components/challenge/challenge.component.html

```diff
@@ -1,42 +1,70 @@
+                                          {{challenge['title']}}
+                                      </strong>
+                                  </span>
+                                  &nbsp;
+                                  <span *ngIf="isChallengeHost">
+                                      <a style="padding-right: 4px;" class="pointer" (click)="editChallengeTitle()">
+                                          <i class="fa fa-pencil" aria-hidden="true"></i>
+                                      </a>
+                                      <a class="pointer" (click)="deleteChallenge()">
+                                          <i class="fa fa-trash red-text" aria-hidden="true"></i>
+                                      </a>
+                                  </span>
+                                  <br>
+                                  <span style="line-height: 1.5" *ngIf="challenge.creator.team_url != ''"><strong class="text-light-black fw-semibold fs-15">Organized by:</strong><a class="orange-text" href="{{challenge['creator']['team_url']}}" target="_blank"> {{challenge['creator']['team_name']}} </a></span>
+                                  <span style="line-height: 1.5" *ngIf="challenge.creator.team_url == ''"><strong class="text-light-black fw-semibold fs-15">Organized by:</strong> {{challenge['creator']['team_name']}} </span>
+
+                                  <br>
+                                  <span style="line-height: 1.5" *ngIf="isChallengeHost">
+                                      <span>
+                                          <a class="pointer" (click)="togglePublishChallengeState()">
+                                              <strong class="text-light-black fw-semibold fs-15">{{publishChallenge.state}} </strong>
+                                              <i class="{{publishChallenge.icon}}" aria-hidden="true"></i>
+                                          </a>
+                                      </span>
+                                  </span>
+                              </p>
+                          </div>
+                      </div>
+                      <div class="col-lg-3 col-md-3 align-right">
+                          <button class="stars btn-waves-effect grad-rec-btn grad-btn-dark ev-btn-dark fw-light fs-14" type="submit" [class.is-clickable]="isLoggedIn" (click)="starToggle(challenge['id'])"><i class="fa fa-star star"></i> {{challenge.data}}&nbsp;&nbsp;{{stars['count']}}</button>
+                      </div>
+                  </div>
+                  <div class="row rm-row-bot card-tab-row">
+                      <div class="col-sm-12">
+                          <ul class="inline-list rm-gut">
+                              <li><a [class.selected]="localRouter.url.endsWith('overview')" [routerLink]="['./overview']" class="text-light-black fw-semibold fs-15"><i class="fa fa-info"></i> Overview</a></li>
```

Comment:
You could have made it readable rather than writing everything in one line.

Reply:
@Shekharrajak done